src/main/java/com/github/jolangbein/RandomTypoGenerator.java

```diff
@@ -10,7 +11,7 @@
     private static final int HIGHEST_ASCII_CHAR = 126;
     private static final int LOWEST_ASCI_CHAR = 33;
 
-    private Random random = new Random();
+    private final Random random = new SecureRandom();
```

Comment:
Nice! Generating typos cannot be taken lightly, so we must use cryptographcally secure random numbers here. Good think! Would you like to update the README file so we can brag about this more publicly?

Reply:
Not sure if we should brag about it. In my opinion, everybody deserves the right/possibility/freedom to have secure typos by secure random numbers!